Documentation/README_SpatialAwareness.md

```diff
@@ -0,0 +1,6 @@
+# Spatial Mapping
+![Spatial Mapping](/External/ReadMeImages/SpatialAwareness/MRTK_SpatialAwareness_Main.png)
+
+## How to use Spatial Mapping
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Quis ipsum suspendisse ultrices gravida. Risus commodo viverra maecenas accumsan lacus vel facilisis. 
```

Comment:
I think we should probably delete this file, or add a 'coming' soon instead of the filler text. (Probably delete? Coming soon isn't usually very helpful)

Reply:
Let’s delete for now. Or if someone can add simple description about the example scene, it would be great.